2024-10-01 20:28:20.980 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-10-01 20:28:20.982 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration alexa_media which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-10-01 20:28:20.983 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration extended_openai_conversation which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-10-01 20:28:20.985 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration auto_backup which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-10-01 20:51:39.454 WARNING (Thread-7) [pychromecast.socket_client] [Living Room TV(192.168.0.22):8009] Heartbeat timeout, resetting connection
2024-10-01 20:52:09.468 ERROR (Thread-7) [pychromecast.socket_client] [Living Room TV(192.168.0.22):8009] Failed to connect to service MDNSServiceInfo(name='BRAVIA-4K-GB-ATV3-6949676269c618f17d4c43768c7d3238._googlecast._tcp.local.'), retrying in 5.0s
2024-10-01 20:54:47.625 WARNING (MainThread) [async_upnp_client.profiles.profile] Failed (re-)subscribing to: uuid:6d4971a6-802e-11ef-8000-800000000000, reason: UpnpConnectionError('ClientConnectorError(ConnectionKey(host=\'192.168.0.22\', port=52323, is_ssl=False, ssl=True, proxy=None, proxy_auth=None, proxy_headers_hash=2955182482271169906), OSError(113, "Connect call failed (\'192.168.0.22\', 52323)"))', None)
2024-10-01 20:54:56.842 WARNING (MainThread) [async_upnp_client.profiles.profile] Failed (re-)subscribing to: uuid:6d4b2078-802e-11ef-8000-800000000000, reason: UpnpConnectionError('ClientConnectorError(ConnectionKey(host=\'192.168.0.22\', port=52323, is_ssl=False, ssl=True, proxy=None, proxy_auth=None, proxy_headers_hash=2955182482271169906), OSError(113, "Connect call failed (\'192.168.0.22\', 52323)"))', None)
2024-10-01 20:54:58.118 WARNING (MainThread) [homeassistant.components.media_player] Updating dlna_dmr media_player took longer than the scheduled update interval 0:00:10
2024-10-01 20:54:58.119 WARNING (MainThread) [homeassistant.helpers.entity] Update of media_player.living_room_tv_2 is taking over 10 seconds
2024-10-01 21:31:17.620 WARNING (Thread-7) [pychromecast.socket_client] [Living Room TV(192.168.0.22):8009] Heartbeat timeout, resetting connection
2024-10-01 21:31:47.657 ERROR (Thread-7) [pychromecast.socket_client] [Living Room TV(192.168.0.22):8009] Failed to connect to service MDNSServiceInfo(name='BRAVIA-4K-GB-ATV3-6949676269c618f17d4c43768c7d3238._googlecast._tcp.local.'), retrying in 5.0s
2024-10-01 21:57:13.180 ERROR (Thread-7) [pychromecast.socket_client] [Living Room TV(192.168.0.22):8009] Failed to connect. No retries.
